app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -274,14 +275,6 @@ const styles = StyleSheet.create({
     marginBottom: 0
   },
 
-  addFunds: {
-    minWidth: '180px',
-    width: 'auto',
-    marginTop: 0,
```

Comment:
the two properties above are specified with `browserButton_default` and `browserButton`